cogs/file_cog.py

```diff
@@ -48,6 +53,29 @@ async def save_jpg_with_target_size(
 class FileCog(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
+        self.active_pdf = pd.DataFrame(
+            {
+                "Message_Id": pd.Series(dtype="int"),
+                "End_Time": pd.Series(dtype="datetime64[ns, America/Toronto]"),
+            }
+        )
```

Comment:
I don't think pandas is necessary for this. We should only add deps if we need to, and I think you can accomplish all of this using lists and dicts without too much extra effort.